moveit_setup_assistant/src/widgets/controller_edit_widget.cpp

```diff
@@ -195,12 +195,11 @@ void ControllerEditWidget::loadControllersTypesComboBox()
     return;
   has_loaded_ = true;
 
-  const std::array<std::string, 9> default_types = {
+  const std::array<std::string, 8> default_types = {
     "effort_controllers/JointTrajectoryController",   "effort_controllers/JointPositionController",
     "effort_controllers/JointVelocityController",     "effort_controllers/JointEffortController",
-    "joint_state_controller/JointStateController",    "position_controllers/JointPositionController",
-    "position_controllers/JointTrajectoryController", "velocity_controllers/JointTrajectoryController",
-    "velocity_controllers/JointvelocityController"
+    "position_controllers/JointPositionController",   "position_controllers/JointTrajectoryController",
+    "velocity_controllers/JointTrajectoryController", "velocity_controllers/JointvelocityController"
```

Comment:
i don't think this is a good work-around for a bug

Reply:
Not a work around, we already add the joint state controllers by default  and it doesnt need the user to add anything to it -it only has a publish rate, right?-. Why would we be adding it to the types of the new conrollers?

Reply:
"we already add the joint state controllers by default"

you mean when that button is pushed? i don't like that there is a hidden controller, we should remove that behavior and require that it be manually added

Reply:
correct, it should not change